### PR TITLE
Support markdown for about link COUNTRY=cambodia

### LINKED ISSUE #588: Make the 'about' page content configurable
I want to be able to configure the 'about' page to read markup content from a local file and display it in a modal, or to be able to configure what external web page is launched when a user clicks on 'about'. 

We could look at how the 'content_path' attribute in layer configuration as an example / potential reuse. The screenshot below is from Myanmar where multiple layers have a 'content_path' attribute, which is accessed by clicking on the info icon in the legend after activating a layer.

Screen Shot 2022-10-11 at 21 29 54

Screen Shot 2022-10-11 at 21 28 17

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
This PR closes #588 

**Changes**
- The Dialog modal was decoupled into a generic component to be used by both layerpreview and about page.
- Markdown information is loaded on a click event rather than a component render.
- The path of the markdown file is specified within the **aboutPath** field within the prism.json file. If the parameter is not found, the link will be the innovation.wfp.org.

### CODE REVIEW

File: src/config/myanmar/prism.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "country": "Myanmar",
-  "WFPAuthRequired": true,
+  "WFPAuthRequired": false,
```
Comment (ericboucher) [inline src/config/myanmar/prism.json:3]: Did you mean to deactivate auhentication?

Comment (wadhwamatic) [reply src/config/myanmar/prism.json]: I removed this which was probably just for testing. We'll use this feature in Cambodia first

**Comment by wadhwamatic** [comment]
This looks good from my side. We'll adjust the content with input from the Cambodia team but it's looking good. Over to @ericboucher for final review

File: public/data/cambodia/about.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Platform for Real-Time Impact and Situation Monitoring (PRISM) is a web-based platform which supports the National Committee for Disaster Management (NCDM), other government agencies and partners to rapidly assess climate hazard risks and their impact, to inform disaster risk management and social protection programs. PRISM simplifies the integration of geospatial data on hazards, such as droughts and floods, along with information on socioeconomic vulnerability (IDPoor) to present decision makers with actionable information on vulnerable populations exposed to hazards.
+
+## NCDM contact info
+
+NCDM's public webiste is accessible at: <a href="http://ncdm.gov.kh/" target="_blank">http://ncdm.gov.kh/</a>
+The general NCDM email address is: <a href="mailto:administration@ncdm.gov.kh" target="_blank">administration@ncdm.gov.kh</a>  
+
+
+<img src="data/cambodia/ncdm_l3.png" width="200" height="200">
```
Comment (ericboucher) [inline public/data/cambodia/about.md:11]: @JorgeMartinezG can you reduce the size (in mb) of the image? It will help it load faster

Comment (wadhwamatic) [reply public/data/cambodia/about.md:11]: > @JorgeMartinezG can you reduce the size (in mb) of the image? It will help it load faster

done

**Comment by JorgeMartinezG** [comment]
I removed the changes from myanmar config, since cambodia is used as example country

**Comment by wadhwamatic** [comment]
> I removed the changes from myanmar config, since cambodia is used as example country

@JorgeMartinezG - I already did that last week =)

**Review by ericboucher — APPROVED** [review]

**Pull request merged**